packages/core/src/browser/frontend-application-config-provider.spec.ts

```diff
@@ -0,0 +1,44 @@
+/********************************************************************************
+ * Copyright (C) 2021 Ericsson and others.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License v. 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * This Source Code may also be made available under the following Secondary
+ * Licenses when the conditions for such availability set forth in the Eclipse
+ * Public License v. 2.0 are satisfied: GNU General Public License, version 2
+ * with the GNU Classpath Exception which is available at
+ * https://www.gnu.org/software/classpath/license.html.
+ *
+ * SPDX-License-Identifier: EPL-2.0 OR GPL-2.0 WITH Classpath-exception-2.0
+ ********************************************************************************/
+
+import { enableJSDOM } from '../browser/test/jsdom';
+let disableJSDOM = enableJSDOM();
+
+import { expect } from 'chai';
+import { FrontendApplicationConfigProvider } from './frontend-application-config-provider';
+
+disableJSDOM();
+
+describe('FrontendApplicationConfigProvider', function (): void {
+
+    before(() => disableJSDOM = enableJSDOM());
+    after(() => disableJSDOM());
+
+    it('set should use defaults', function (): void {
```

Comment:
minor: for consistency with the rest of the framework we should probably align the naming to `it('should ...')`:

```suggestion
    it('should use the defaults when calling `set`', function (): void {
```

I am wondering if using the defaults is the intention of the test, perhaps it is more that it supports partial updates?:

```suggestion
    it('should support partial updates when invoking `set`, function (): void {
```